peer/pendingheap/heap.go

```diff
@@ -166,6 +172,28 @@ func (ph *pendingHeap) pushPeer(ps *peerScore) {
 	heap.Push(ph, ps)
 }
 
+// pushPeerRandom must be called in the context of a lock.
+//
+// It inserts a peer randomly into the heap among equally scored peers. This is
+// expected to be called only by Add.
```

Comment:
Nit, second sentence should be first. Describe purpose, behavior, then invariants.